backend/app/Utils/HostIndicationsProvider.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+
+namespace App\Utils;
+
+class HostIndicationsProvider
```

Comment:
Rename it to service (HostIndicationsService), get rid of static methods, and use laravel service container instead. I mean inject service to constructor of your action